components/layout_2020/fragment_tree/base_fragment.rs

```diff
@@ -75,31 +75,33 @@ impl From<BaseFragmentInfo> for BaseFragment {
 bitflags! {
     /// Flags used to track various information about a DOM node during layout.
     #[derive(Clone, Copy, Debug)]
-    pub(crate) struct FragmentFlags: u8 {
+    pub(crate) struct FragmentFlags: u16 {
         /// Whether or not the node that created this fragment is a `<body>` element on an HTML document.
         const IS_BODY_ELEMENT_OF_HTML_ELEMENT_ROOT = 1 << 0;
         /// Whether or not the node that created this Fragment is a `<br>` element.
         const IS_BR_ELEMENT = 1 << 1;
         /// Whether or not this Fragment is a flex item or a grid item.
-        const IS_FLEX_OR_GRID_ITEM = 1 << 2;
+        const IS_INPUT_ELEMENT = 1 << 2;
```

Comment:
```suggestion
        const IS_TEXT_CONTROL = 1 << 2;
```
For naming consistency. Also "text control" stands for both `<input>` and `<textarea>`